comdb2/dbapi2.py

```diff
@@ -303,17 +350,27 @@
 Comdb2's native placeholder format is ``@name``, but that cannot be used by
 this module because it's not an acceptable `DB-API 2.0 placeholder style
 <https://www.python.org/dev/peps/pep-0249/#paramstyle>`_.  This module uses
-``pyformat`` because it is the only DB-API 2.0 paramstyle that we can translate
-into Comdb2's placeholder format without needing a SQL parser.
+``pyformat`` for named parameters because it is the only DB-API 2.0 paramstyle
+that we can translate into Comdb2's placeholder format without needing
+a SQL parser. This module also supports the ``qmark`` parameter style for
+binding parameters positionally.
 
 Note:
     An int value is bound as ``%(my_int)s``, not as ``%(my_int)d`` - the last
     character is always ``s``.
 
 Note:
-    Because SQL strings for this module use the ``pyformat`` placeholder style,
-    any literal ``%`` characters in a query must be escaped by doubling them.
-    ``WHERE name like 'M%'`` becomes ``WHERE name LIKE 'M%%'``.
+    When binding parameters by name, any ``%`` sign is recognized as the start
+    of a ``pyformat`` style placeholder, and so any literal ``%`` characters in
+    a query must be escaped by doubling. ``WHERE name like 'M%'`` becomes
+    ``WHERE name LIKE 'M%%'``. This is not necessary when binding parameters
```

Comment:
Not only is it not necessary, but should not be done. Let's word this in a more restricted way.